### PR TITLE
PUD-1655 Weekly recalls new report Front End

### PR DESCRIPTION
**Pull request by AndyNewton**
PUD-1655 Weekly recalls new report Front End
Better for you to review this - rather than me just merge - IMO @jonnywyatt though maybe a bit late for this week now?

### CODE REVIEW

File: fake-manage-recalls-api/stubs/__files/get-weekly-recalls-new.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "category": "WEEKLY_RECALLS_NEW",
+  "mimeType": "text/csv",
+  "fileName": "weekly-recalls-new.csv",
+  "content": "CUSTODY_TYPE_DESCRIPTION,CUSTODY_TYPE_AT_TIME_OF_RECALL_DESCRIPTION,FAMILY_NAME,FIRST_NAMES,NOMS_ID,PRISON_NUMBER\nNot in custody,In custody,Surname,Firstname Billy Bob,A1234AA,A123456\n"
```
Comment (jonnywyatt) [inline fake-manage-recalls-api/stubs/__files/get-weekly-recalls-new.json:5]: This is nitpicking, but everywhere else I've seen, we refer to these as `NOMS number` and `Booking number`

File: server/views/pages/reports.njk [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                        'data-qa': 'error-list'
+                    }
+                }) }}
+            {% endif %}
+            <h1 class='govuk-heading-l'>Reports</h1>
+
+            <p class='govuk-body-l'>Click on a report to download it.</p>
+            <div{% if errors.error_WEEKLY_RECALLS_NEW %} id='downloadLinkError'{% endif %}>
+            {{ documentDownloadLink({
+                linkText: "Weekly Recalls New",
```
Comment (jonnywyatt) [inline server/views/pages/reports.njk:26]: "Weekly New Recalls" reads a bit better?

File: server/clients/manageRecallsApiClient.ts [hunk 3]
```diff
@@ -303,6 +304,10 @@ export function getServiceMetrics(token: string): Promise<StatisticsSummary> {
   return restClient(token).get<StatisticsSummary>({ path: '/statistics/summary' })
 }
 
+export function getWeeklyRecallsNew(token: string): Promise<GetReportResponse> {
```
Comment (jonnywyatt) [inline server/clients/manageRecallsApiClient.ts:307]: Can't believe this is me asking you this, but shouldn't you add a PACT?

**Review by jonnywyatt — CHANGES_REQUESTED** [review]

File: fake-manage-recalls-api/stubs/__files/get-weekly-recalls-new.json (hunk 1 above)
Comment (AndyNewton) [reply fake-manage-recalls-api/stubs/__files/get-weekly-recalls-new.json:5]: These column heading for the report are dictated by the story - to match the PPUD terms.

File: server/views/pages/reports.njk (hunk 2 above)
Comment (AndyNewton) [reply server/views/pages/reports.njk:26]: Yes; we can change of course; but this is the report name as per the story.  In time we'll get feedback from Product team of course.

File: server/clients/manageRecallsApiClient.ts (hunk 3 above)
Comment (AndyNewton) [reply server/clients/manageRecallsApiClient.ts:307]: I will look; needs to be added on the API first and isn't there yet.  And given this is almost a 'prototype' feature we may choose to skip it.

**Review by jonnywyatt — APPROVED** [review]

**Pull request merged**